packages/neuron-ui/src/components/Overview/index.tsx

```diff
@@ -134,17 +137,26 @@ const Overview = () => {
           typeLabel = `${t(`overview.${genTypeLabel(type, status)}`)}`
           amount = `${type === 'receive' ? '+' : '-'}${nftFormatter(data)}`
         } else if (item.sudtInfo?.sUDT) {
-          const type = +item.sudtInfo.amount <= 0 ? 'send' : 'receive'
-          typeLabel = `UDT ${t(`overview.${genTypeLabel(type, status)}`)}`
-
+          if (item.type === 'create' || item.type === 'destroy') {
+            typeLabel = `${t(`overview.${item.type}`, { name: item.sudtInfo.sUDT.tokenName || 'UDT' })}`
+          } else {
+            const type = +item.sudtInfo.amount <= 0 ? 'send' : 'receive'
+            typeLabel = `UDT ${t(`overview.${genTypeLabel(type, status)}`)}`
+          }
           if (item.sudtInfo.sUDT.decimal) {
             amount = `${sUDTAmountFormatter(sudtValueToAmount(item.sudtInfo.amount, item.sudtInfo.sUDT.decimal))} ${
               item.sudtInfo.sUDT.symbol
             }`
           }
         } else {
           amount = `${shannonToCKBFormatter(item.value)} CKB`
-          typeLabel = item.nervosDao ? 'Nervos DAO' : t(`overview.${genTypeLabel(item.type, status)}`)
+          if (item.type === 'create' || item.type === 'destroy') {
+            if (item.assetAccountType === 'CKB') {
+              typeLabel = `${t(`overview.${item.type}`, { name: 'CKB' })}`
+            }
```

Comment:
This else arm is not covered, maybe we can name it `Unknown`(it's reserved) so when `create/destroy Unknown Asset Account` does appear we can ensure this arm is triggered.